scripts/development/create_databases.sh

```
@@ -22,5 +22,5 @@ if [ -f $gamesdb ]; then
 fi
 
 # create db, tables
-echo "CREATE TABLE players (id INT PRIMARY_KEY, name VARCHAR(20), password VARCHAR(50), email VARCHAR(50), rating real default 1000, boost real default 750, ratedgames int default 0, maxrating real default 1000, ratingage real default 0, ratingbase int default 0, unrated int default 0, isbot int default 0, fatigue text default '{}', is_admin int default 0, is_mod int default 0, is_gagged int default 0, is_banned int default 0, participation_rating int default 1000);" | sqlite3 $playersdb
+echo "CREATE TABLE players (id INTEGER PRIMARY KEY, name VARCHAR(20), password VARCHAR(50), email VARCHAR(50), rating real default 1000, boost real default 750, ratedgames int default 0, maxrating real default 1000, ratingage real default 0, ratingbase int default 0, unrated int default 0, isbot int default 0, fatigue text default '{}', is_admin int default 0, is_mod int default 0, is_gagged int default 0, is_banned int default 0, participation_rating int default 1000);" | sqlite3 $playersdb
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)